purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -504,7 +504,7 @@ class PurchasesTest {
     fun passesUpErrors() {
         var errorCalled = false
         val storeProduct = createStoreProductWithoutOffers()
-        purchases.purchaseProductWithOption(
+        purchases.purchaseProductOption(
```

Comment:
Hmm should some of these be `purchaseProductOptionWith`?

Reply:
yeah great catch. i did refactor > rename in such a way that they ALL good renamed each time haha so updating usages didn't work well